packages/createHash/src/index.ts

```diff
@@ -1,7 +1,13 @@
 import { isNodeLike } from "@apollo/utils.isnodelike";
 
 export function createHash(kind: string): import("crypto").Hash {
-  if (isNodeLike) {
+  // Some Node-like environments (like next.js Turbopack) apparently
+  // don't have module.require, so double-check before we call it.
+  // (But don't the value of isNodeLike because other logic depends on it,
```

Comment:
Something missing here, is this it?
```suggestion
  // (But don't change the value of isNodeLike because other logic depends on it,
```